packages/core/quri_parts/core/estimator/gradient.py

```diff
@@ -62,7 +64,7 @@ def numerical_gradient_estimates(
     Returns:
         The estimated values (can be accessed with :attr:`.values`) with errors
         of estimation (can be accessed with :attr:`.error_matrix`). Currently,
-        :attr:`.error_matrix` returns `None`.
+        non-diagonal elements of :attr:`.error_matrix` are filled with zeros.
```

Comment:
Saying "are filled with zeros" sounds to me like "we don't know the exact values but we put zeros at the moment", but I guess the covariances are exactly zero in this case since each element is estimated indepedently.

Reply:
I was a bit confused but, as you guessed, the covariance are exactly zero.
So I've removed the sentence.